apps/analog-app/vite.config.ts

```diff
@@ -32,6 +32,9 @@ export default defineConfig(({ mode }) => {
         apiPrefix: 'api',
         prerender: {
           routes: ['/', '/cart'],
+          sitemap: {
+            domain: 'example.com/',
```

Comment:
Use the custom base URL for this example

```suggestion
            domain: process.env['VITE_ANALOG_PUBLIC_BASE_URL'],
```